package/MDAnalysis/topology/tables.py

```diff
@@ -394,3 +394,37 @@ def kv2dict(s, convertor=str):
           113: 'Nh', 114: 'Fl', 115: 'Mc', 116: 'Lv', 117: 'Ts', 118: 'Og'}
 
 SYMB2Z = {v:k for k, v in Z2SYMB.items()}
+
+# Conversion between SYBYL atom types and corresponding elements
+# Tripos MOL2 file format: https://zhanglab.ccmb.med.umich.edu/DockRMSD/mol2.pdf
```

Comment:
That PDF doesn't seem to point to anything? (at least on my end)

Reply:
Odd, I can't access it anymore... Does the following work? [http://chemyang.ccnu.edu.cn/ccb/server/AIMMS/mol2.pdf](http://chemyang.ccnu.edu.cn/ccb/server/AIMMS/mol2.pdf)

Reply:
Yeah that works for now. I think that technically the sybyl manual is copyrighted, not sure how to go about this 🤔 @orbeckst any ideas?

Reply:
See if it's archived in the internet archive and link to it there. 

WebCITE does not accept new submissions anymore, I think.

Ideally, *someone* puts it on zenodo or figshare so that we have a DOI. Perhaps ask MolSSI to archive it as an important document.

Reply:
@RMeli For the sake of getting this merged quickly, how about we raise an issue about adding details about the mol2 format manual, remove the link for now and then we can go bug the right person so we can link to it in the most appropriate way?

Reply:
I don't think the link is inappropriate at all, it resolves — and that's all a user cares about. As long as someone else is hosting it...

Long-term a better/archive-grade link would be good. Raise an issue for that but keep the link!

Reply:
I mean, put in the working link from https://github.com/MDAnalysis/mdanalysis/pull/3063#discussion_r687938122

Reply:
👍🏽 sounds good!